rubin_changelog/changelog.py

```diff
@@ -129,9 +129,12 @@ def _get_package_repos(self, products: SortedList) -> SortedDict:
         repos = gh.get_repos()
         del gh
         repo_list = SortedList()
-        for product in products:
-            if product in repos:
-                repo_list.add(product)
+        # create a lowercase list for GitHub/EUPS package comparison
+        # GitHub will be used for the package name
+        products_lower = [k.lower() for k in products]
```

Comment:
It's slightly more efficient to lookup if you use a set, I think.

But stepping back, couldn't you use repos.yaml to get a comprehensive, authoritative mapping from eups package to GitHub repo?